libs/dh/shared/assets/src/assets/b2c-custom-layouts/reset-password.html

```diff
@@ -58,21 +58,8 @@
 
     <style>
       /** Form **/
-      #cancel {
-        background: initial;
-        color: var(--gray-600);
-        text-transform: initial;
-      }
-
-      #cancel::before {
-        border-right: 2px solid var(--gray-600);
-        border-bottom: 2px solid var(--gray-600);
-        content: "";
-        display: inline-block;
-        height: 10px;
-        width: 10px;
-        margin-right: 8px;
-        transform: rotate(-225deg);
+      button#cancel {
+        display: none;
```

Comment:
question: no more Cancel button?

Reply:
You just close the window or go back :-)